Switching from a `.json` to a `.js` file for default preset configuration

denniscoorn opened the pull request:
A lot of packages nowadays are switching to a regular
.js file for configuration instead of a .json file.
We are following this switch by changing to .js
instead of .json file for our default preset configuration.

---

- [ ] The pull request is complete according to the [Definition of Done](https://sites.google.com/leviy.com/development/development/definition-of-done)
  - [ ] Acceptance criteria in the ticket have been met
  - [ ] Automated tests are written according to the [test plan](https://docs.google.com/document/d/106bwyTS-gIrT7edLW4_5pCl6DSDaZBEMiiyBw1za0R8/edit#heading=h.5ybg7xjjzlkj)
  - [ ] The documentation is up-to-date
  - [ ] Deployment automation includes newly introduced environment variables and credentials are added to the credentials.fact file of all servers
- [ ] The pull request has a descriptive title that describes what has changed, and provides enough context for the changelog
- [ ] When applicable, the pull request title starts with the issue number

Commit:
A lot of packages nowadays are switching to a regular
.js file for configuration instead of a .json file.
We are following this switch by changing to .js
instead of .json file for our default preset configuration.

christiaan commented on .github/workflows/npmpublish.yml:
```diff
@@ -14,6 +14,7 @@ jobs:
           node-version: 12
           registry-url: https://npm.pkg.github.com/
           scope: '@leviy'
+      - run: npm version ${{github.event.release.name}}
```
A disadvantage I can imagine of this change is that the version in the package.json of a checkout of this repository will still have the old version in it. Whilst the version on pkg.github.com has a different version as that one is bumped by this command.

A quick fix for this that I can think of is to specify version `0.0` in the package.json so that it's clear that it is incorrect.

denniscoorn replied:
Correct. I also thought about the "`0.0`" solution. Maybe I'm going to change, now you mentioned it.

denniscoorn replied:
Done, let's see if this will work 😄

Commit:
Replaced version number with "0.0.0" to illustrate that this version
number is not actually used. Version number will be set during a
GitHub Action before publishing the package to the GitHub Registry.
